src/commands/mathspace/mathspaceCommands.js

```diff
@@ -12,6 +12,42 @@ CharCmds['*'] = LatexCmds.times;
 // Patched latex for % symbol, it should not contain \\ in the beginning.
 LatexCmds['%'] = bind(NonSymbolaSymbol, '%', '%');
 
+var MixedFraction = 
```

Comment:
Ideally I would like to see a definition consistent with what has come before.

**opinion** The additional `var` here is confusing to me.

**question** How is the `P(...)` definition different to the `bind` definition?

Reply:
withdrawn: I now see this is consistent with what is already here (below)